modules/home/assets/js/icons/side-bar-cta-icon.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
```

Comment:
I think we can delete this file, because I don't think that we are using it.

Reply:
fixed